### DISCUSSION TITLE
Analog render.injectScripts in nuxt 3?

### DISCUSSION THREAD

**Discussion by popovdk** [nuxt/nuxt#16953] [Questions]
Hey, please tell me if there is an analogue of the parameter render.injectScripts from nuxt 2 to 3 version? Or how to get only HTML without JS in other ways?

We are working on performance, we want to abort the scripts provided from Nuxt and fully control the execution of the client JS.

**Comment by gplusdotgr** [comment]
@danielroe is working on something like that.

**Reply by danielroe** [reply]
You can follow the feature request discussion in https://github.com/nuxt/framework/issues/7156.
